themes/ddbasic/sass/module/_search.scss

```diff
@@ -206,6 +206,17 @@ input#edit-search-block-form--2.spinner {
   .reserve-button {
     @extend %button-action;
   }
+
+  .find-items-button {
+    @extend %button-theme;
+
+    float: left;
+  }
+
+  .bookmark-button {
```

Comment:
To align bookmark button with find the button. If didn't add this style for button bookmark, the button is higher than find the item button.